doc/operator/deploy-operator.md

```diff
@@ -0,0 +1,185 @@
+# Observatorium Operator
+![Observatorium](https://avatars0.githubusercontent.com/u/51818702?s=100&v=4sanitize=true)
+
+## Prelude
+
+Check the following resources for more information about Observatorium:
+* [Documentation repository](https://github.com/observatorium/docs/)
+* [Observatorium API](https://github.com/observatorium/observatorium/)
+* [Observatorium configuration repository](https://github.com/observatorium/configuration)
+* [Locutus - The framework which the operator is based on](https://github.com/brancz/locutus)
+
+
+## How to deploy - Kubernetes and OpenShift
+
+In order to ease the installation of Observatorium, an operator is available.
+
+### Prerequisites
+
+#### Create Namespaces
+```shell script
+kubectl create namespace minio
+kubectl create namespace observatorium
+```
+
+
+#### S3 storage endpoint and secret
+For **testing purposes** you may use [minio](https://github.com/minio/minio) as describe below.
+```shell script
+kubectl apply -f https://raw.githubusercontent.com/observatorium/configuration/master/environments/dev/manifests/minio-secret.yaml
+kubectl apply -f https://raw.githubusercontent.com/observatorium/configuration/master/environments/dev/manifests/minio-pvc.yaml
+kubectl apply -f https://raw.githubusercontent.com/observatorium/configuration/master/environments/dev/manifests/minio-deployment.yaml
+kubectl apply -f https://raw.githubusercontent.com/observatorium/configuration/master/environments/dev/manifests/minio-service.yaml
+```
+
+### Deployment
+
+#### Prometheus CRDs - Kubernetes Only
+```shell script
+kubectl apply -f https://raw.githubusercontent.com/coreos/kube-prometheus/master/manifests/setup/prometheus-operator-0servicemonitorCustomResourceDefinition.yaml
+kubectl apply -f https://raw.githubusercontent.com/coreos/kube-prometheus/master/manifests/setup/prometheus-operator-0prometheusruleCustomResourceDefinition.yaml
+```
+
+#### RBAC Configuration
+```shell script
+kubectl apply -f https://raw.githubusercontent.com/observatorium/configuration/master/deploy/cluster_role.yaml
+kubectl apply -f https://raw.githubusercontent.com/observatorium/configuration/master/deploy/cluster_role_binding.yaml
+```
+
+#### Deploy Observatorium CRD and Operator
+```shell script
+kubectl apply -f https://raw.githubusercontent.com/observatorium/configuration/master/deploy/crds/core.observatorium.io_observatoria.yaml
+# Create the operator while forcing Kubernetes / OpenShift to download the image.  
+curl https://raw.githubusercontent.com/observatorium/configuration/master/deploy/operator.yaml | \
+sed 's/imagePullPolicy\: IfNotPresent/imagePullPolicy\: Always/g' > observatorium-operator.yaml && \
```

Comment:
that's only for development purposes, I wouldn't have that in these docs

Reply:
I think it brings value to the reader (so they can try it out).
Would a better reflection of "this is just for you to experiment with the operator" be in place here?

Reply:
@brancz ping

Reply:
People tend to copy what they find, I would prefer to rather keep this separate in a development section.